### DISCUSSION TITLE
Organization variable is empty in pull request from non-member user

### DISCUSSION THREAD

**Discussion by FreePhoenix888** [community/community#61170] [Actions]
### Select Topic Area

Bug

### Body

Our team has reusable workflow: https://github.com/deep-foundation/workflows/blob/main/.github/workflows/npm-build.yml which we use in every our repository. We have set `NPM_BUILD_COMMAND` organization variable to `npm run package:build`.
Here is how we use that workflow in our `capacitor-device` repository: https://github.com/deep-foundation/capacitor-device/blob/main/.github/workflows/npm-build.yml
and as you can see it works fine: https://github.com/deep-foundation/capacitor-device/actions/runs/5586454496/jobs/10210502676 (take a look at step `Run npm run package:build`

But in this github action run runned for pull-request there is an empty step: https://github.com/deep-foundation/deeplinks/actions/runs/5589943051/jobs/10218980214?pr=193
That is because for some reasons vars.NPM_BUILD_COMMAND here https://github.com/deep-foundation/workflows/blob/main/.github/workflows/npm-build.yml#L9 is empty on pull requests. Why? I have not found information that organization variables are hidden for non-member on pull requests, only about secrets

**Comment by jcs090218** [comment]
I've this exact same issue. :(
